### PR TITLE
WIP - Merge bluetargeting-dev to master

### PR DESCRIPTION
**Pull request by charles474**
For Cerberus v2.0.3 release

### CODE REVIEW

**Review by Shaks — DISMISSED** [review]
lgtm

File: client/src/utils/hooks.js [hunk 1]
```diff
@@ -204,7 +204,9 @@ export const useFetchGroups = () => {
       const {
         tokenParsed: { team_id: teamid },
       } = keycloak;
-      const allUserGroups = keycloak.tokenParsed.groups.join(',');
+      const allUserGroups = isArray(keycloak.tokenParsed.groups)
```
Comment (aliktb) [inline client/src/utils/hooks.js:207]: Just out of curiosity, why would this not be an array?

Comment (charles474) [reply client/src/utils/hooks.js:207]: I'm not really sure why it wouldn't be an array to begin with. My only guess is someone being overly cautious but a good point.

**Comment by charles474** [comment]
Reopened in https://github.com/UKHomeOffice/cop-ui/pull/597

**Pull request closed without merging**